src/Components/Buttons/button.css

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+.btn {
+	font-family: var(--global-text-font--Family);
+	font-weight: 400;
+	border-radius: 10px;
+	cursor: pointer;
+	transition: transform 0.3s ease;
+	margin: 10px 10px 10px 10px;
+}
+
+.btn:hover {
+	transform: translateY(-3px);
+}
+
+.btn:disabled {
+	cursor: not-allowed;
+	transform: translate(0px);
+	opacity: 0.5;
+}
+
+.btn--primary--solid {
+	background-color: var(--global--color--Primary);
+	color: var(--global--color--White);
+	border: none;
+}
+
+.btn--warning--solid {
+	background-color: var(--global--color--Warning);
+	color: var(--global--color--Dark);
+	border: none;
+}
+
+.btn--danger--solid {
+	background-color: var(--global--color--Danger);
+	color: var(--global--color--White);
+	border: none;
+}
+
+.btn--success--solid {
+	background-color: var(--global--color--Success);
+	color: var(--global--color--White);
+	border: none;
+}
+
+.btn--primary--outline {
+	background-color: var(--global--color--White);
+	color: var(--global--color--Primary);
+	border: 2px solid var(--global--color--Primary);
+}
+
+.btn--success--outline {
+	background-color: var(--global--color--White);
+	color: var(--global--color--Success);
+	border: 2px solid var(--global--color--Success);
+}
+
+.btn--warning--outline {
+	background-color: var(--global--color--Dark);
+	color: var(--global--color--Warning);
+	border: 2px solid var(--global--color--Warning);
+}
+
+.btn--danger--outline {
+	background-color: var(--global--color--White);
+	color: var(--global--color--Danger);
+	border: 2px solid var(--global--color--Danger);
+}
+
+.btn--small {
+	padding: 8px 15px;
+	font-size: 16px;
```

Comment:
Define variables for font size in the global CSS file e.g.  `font-size-16: 16px;`